### PR TITLE
Makeattrib fix

### PR DESCRIPTION
**Pull request by bsousi5**
Add rv32imc support and fixing to check for "rtl/arty/hifive" string instead of compare

### CODE REVIEW

**Comment by nategraff-sifive** [comment]
Should this PR be targeting master instead of v201903-branch?

**Comment by bsousi5** [comment]
It should be, trying to remove the PR and repost

**Comment by nategraff-sifive** [comment]
You can edit the target branch without creating a new PR. Just click "Edit" at the top.

**Comment by nategraff-sifive** [comment]
Fixed it.

File: multilib.h++ [hunk 1]
```diff
@@ -25,6 +26,7 @@ std::string arch2abi(std::string arch)
 {
     if (arch == "rv32i")      return "ilp32";
     if (arch == "rv32im")     return "ilp32";
+    if (arch == "rv32imc")     return "ilp32";
```
Comment (nategraff-sifive) [inline multilib.h++]: Just an indentation problem here.

**Pull request merged**